ai_summarizer/models.py

```
@@ -1,16 +1,15 @@
-from __future__ import annotations
 from django.core.exceptions import ValidationError
-import uuid
 
 
 class AISummary(models.Model):
-    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    # Using Django's default BigAutoField (id field is automatically created)
     patient = models.ForeignKey('patients_core.Patient', on_delete=models.CASCADE)
     content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
-    object_id = models.UUIDField()
+    object_id = models.PositiveIntegerField()  # Changed from UUIDField to PositiveIntegerField
```

Ruff (0.12.2)

Line too long (95 > 88)

(E501)